src/lib/swapper/swappers/ThorchainSwapper/utils/getL1quote.ts

```diff
@@ -109,15 +106,15 @@ export const getL1quote = async (
   const getRouteValues = (quote: ThornodeQuoteResponseSuccess, isStreaming: boolean) => ({
     source: isStreaming ? THORCHAIN_STREAM_SWAP_SOURCE : SwapperName.Thorchain,
     quote,
-    // expected receive amount after slippage (no affiliate_fee or liquidity_fee taken out of this value)
-    // TODO: slippage is currently being applied on expected_amount_out which is emit_asset - outbound_fee,
-    //       should slippage actually be applied on emit_asset?
-    expectedAmountOutThorBaseUnit: subtractBasisPointAmount(
-      quote.expected_amount_out,
-      quote.fees.slippage_bps,
-    ),
+    // don't take affiliate fee into account, this will be displayed as a separate line item
+    expectedAmountOutThorBaseUnit: bnOrZero(quote.expected_amount_out)
+      .plus(bnOrZero(quote.fees.affiliate))
+      .toFixed(),
     isStreaming,
     affiliateBps: quote.fees.affiliate === '0' ? '0' : requestedAffiliateBps,
+    // always use TC auto stream quote (0 limit = 5bps - 50bps, sometimes up to 100bps)
+    // see: https://discord.com/channels/838986635756044328/1166265575941619742/1166500062101250100
+    slippageBps: isStreaming ? bn(0) : inputSlippageBps,
```

Comment:
not really used since we aren't displaying slippage for streaming swaps nor modifying the memo, but added to be explicit with the intention in case things change.

Reply:
Do we also want to hide the slippage settings/popover for streaming swaps since it's not applicable?
<img width="331" alt="image" src="https://github.com/shapeshift/web/assets/17035424/a27aeaee-afd7-4d9d-b0f5-af25de32b155">

Reply:
Nevermind, saw this is TODO'd in the PR description, so fine by me 🙏🏽